tutorials/Tutorial 5 - Plotting.ipynb

```diff
@@ -11,12 +11,13 @@
   },
```

Comment:
<p>@mcontisc, did you try using <code>xgi.subhypergraph(H, edges=id_edges_order)</code> ? If you tried that and it didn't work then we need to fix it so that it works simply by passing in a bunch of edges cc. @nwlandry </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ComplexGroupInteractions/xgi/pull/234/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>It works but then it keeps the isolated nodes, and I didn't want them in the plot. Since it is a frozen object, I cannot use the function clean up and that's why I pass the set of nodes as well. </p>

Reply:
<p>Where should I leave this note?</p>

Reply:
Ah sorry for being vague. I just meant "let's write this down somewhere/somehow so that we don't forget".... by which I suppose I mean let's open an issue for it. 

I just opened it #235 .

Reply:
<p>Two thoughts: </p><ul><li>to not keep isolated nodes, you can also do it like this, which is a bit shorter:</li></ul><pre class="ql-syntax" spellcheck="false">order = 2&nbsp;&nbsp;
edges_order = H.edges.filterby('order', order).members()
H_sub = xgi.Hypergraph(edges_order)
</pre><ul><li>when doing this in the past I actually wanted to keep isolated nodes so that I could use the same pos argument to plot different orders, like here https://github.com/maximelucas/plot_simplicial_complex/blob/main/draw_simplicial_complexes.ipynb</li></ul><p>		  Nothing needed for this one, just a thought if we end up providing a helper function to do this</p>

Reply:
<blockquote>"an edge case" </blockquote><p>Good one Leo </p>

Reply:
<p>Thank you @maximelucas : ) I wanted to use subhypergraph to show an additional feature of the library. But if you prefer, I can change !! </p>

Reply:
Aah good idea. In this case I'd say it looks a bit too complicated, so I'd go with the shorter one and showcase subhypeprgraph somewhere else :)

Reply:
I believe subhypergraph is already used in another tutorial. On the other hand, it's not a terribly well documented feature so it'd be good to have it on this one too.

... but having said that, maybe adding it here randomly is not a good solution for the fact that it's poorly documented and we should just go with what's best for this particular case.

Ok @maximelucas you convinced me. @mcontisc please use Max's suggestion instead of subhypergraph (sorry I led you down that path!)

Reply:
Yea that was my thinking too: let's showcase it when it's a natural best solution

Reply:
<p>Done :)</p><p> </p>